build/bakefiles/scintilla.bkl

```diff
@@ -55,6 +54,7 @@
         <define>$(WXUNIV_DEFINE)</define>
         <define>$(DEBUG_DEFINE)</define>
         <define>$(UNICODE_DEFINE)</define>
+        <define>LPEG_LEXER=0</define>
```

Comment:
Out of curiosity, why do we disable this one?

Reply:
When enabled, it needs external lua files that are not part of the scintilla package:
https://github.com/wxWidgets/wxWidgets/blob/06dbd32194b2978b6162d61cbe6237eea7b1b661/src/stc/scintilla/lexers/LexLPeg.cxx#L39-L44

Reply:
FYI, lpeg lexer can be built and loaded separately, so no functionality is really lost if it's not included by default (although it's not going to be statically linked). It uses `LoadLexerLibrary` and it does require a bit of a setup (for example, `lexer.lpeg.home` property may need to be set). I've been using this lexer/approach for many years in my wxiwdgets/wxlua-based project and can provide some additional details if needed.

Another option is to bundle those Lua files, but unfortunately they are (Lua) version dependent and will load a Lua library at run-time, so not going to work without it being present anyway.